taxi/views.py

```diff
@@ -87,3 +89,29 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
```

Comment:
If you have customized queryset you don't need to chose model, it will be anyway overloaded with custom data so you can just delete lane 90